package.json

```diff
@@ -16,7 +16,7 @@
   },
   "scripts": {
     "lint": "standard update",
-    "update": "./update",
+    "update": "./update --max-files-per-batch 1",
```

Comment:
Is 1 really the right number? Shouldn't we balance out IO and hash computations?

Reply:
Ah you wrote that in the issue description. Maybe it's because the files were recently written to disk and are still in the cache or something...

Reply:
I don't think so - at least in CI the running time did not change significantly. I think that reading files from disk just does not take that much time compared to hashing, at least on a fast disk. This is also why it's so much faster without `--update-hashes`.

In any case, by default there's no limit so, if there is any speed decrease, you won't get it when running it locally.